cmd/eskip/mediadefaults.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
this code feels a bit like an overkill. Wouldn't it be simpler as a single function? E.g:

```
func defaultMedia(cmd command, in, out *medium) (*medium, *medium, error) {
    var err error
    switch cmd {
    case check, print:
        if in == nil {
            in, err = processEtcdArgs(defaultEtcdUrls, defaultEtcdPrefix)
        }
    case upsert, reset, delete:
        if out == nil {
            out, err = processEtcdArgs(defaultEtcdUrls, defaultEtcdPrefix)
        }
    }

    return in, out, err
}
```